org.jacoco.core.test/src/org/jacoco/core/internal/analysis/InstructionTest.java

```diff
@@ -164,11 +162,29 @@ public void replaceBranches_should_calculate_coverage_on_new_branches() {
 		Instruction i1 = new Instruction(1);
 		Instruction i2 = new Instruction(2);
 		Instruction i3 = new Instruction(3);
```

Comment:
At least this violates `Instruction` contract - see https://github.com/jacoco/jacoco/pull/1813/files#r1883024911